wagtail/admin/wagtail_hooks.py

```diff
@@ -863,7 +867,11 @@ def is_shown(self, request):
 
 class AgingPagesReportMenuItem(MenuItem):
     def is_shown(self, request):
-        return getattr(settings, "WAGTAIL_AGING_PAGES_ENABLED", True)
+        return getattr(
+            settings, "WAGTAIL_WORKFLOW_ENABLED", True
```

Comment:
The aging pages report isn't related to workflows, so I think this part of the check can be removed, just leaving the PagePermissionPolicy check.

Reply:
Apologies I missed that, corrected now!!